src/main/java/seedu/duke/storage/Storage.java

```diff
@@ -1,7 +1,139 @@
 package seedu.duke.storage;
 
+import seedu.duke.record.Patient;
+import seedu.duke.storage.PatientList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Storage {
-    public Storage() {
 
+    private String appointmentListSaveLocation;
+    private String patientListSaveLocation;
+    private PatientList patientList = new PatientList();
+    private AppointmentList appointmentList = new AppointmentList();
+
+    private static final String APPOINTMENT_LIST_SAVE_DIRECTORY = "saves";
+    private static final String APPOINTMENT_LIST_SAVE_FILEPATH = "saves/appointments.txt";
+    private static final String PATIENT_LIST_SAVE_DIRECTORY = "saves";
```

Comment:
can declare SAVE_DIRECTORY = "saves" instead of separating directories for appointment and patient

Reply:
good point, will make the change